### DISCUSSION TITLE
Specific only necessary fields when queryBasicTraces in TraceQueryEsDAO

### DISCUSSION THREAD

**Discussion by dogblues** [apache/skywalking#11411] [General]
TraceQueryEsDAO query whole segment doc when query basic traces. When single segment is very complicated and the data_binary is huge, query baisc traces maybe slow. Actually we do not need the data_binary. So we can specific only necessary fields when query es, which is done in JDBCTraceQueryDAO.

**Comment by wu-sheng** [comment]
Yes, it makes sense. Submit a pull request if you want.

**Reply by dogblues** [reply]
Okay i'll do it asap

**Comment by wu-sheng** [comment]
Ref https://github.com/apache/skywalking/pull/11419 for more details.
